examples/exercises/plot_cv_diabetes.py

```diff
@@ -4,10 +4,6 @@
 ===============================================
 
 A tutorial exercise which uses cross-validation with linear models.
-
-This exercise is used in the :ref:`cv_estimators_tut` part of the
-:ref:`model_selection_tut` section of the :ref:`stat_learn_tut_index`.
```

Comment:
This references the old `doc/tutorial/statistical_inference/model_selection.rst` (you can see it in #29104).
I think the user guide section https://scikit-learn.org/stable/modules/grid_search.html#model-specific-cross-validation pretty much gives similar information as the old tutorial. I would add a link to this example there, but I am a little confused about what this example is trying to say? That `GridSearchCV` cannot be trusted?!